src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -439,6 +445,7 @@ public static String jobConfig(String jobKey, String cfgName) {
 
             this.put(IDENTITY_CERT_YEAR_ADDENDUM, "5");
             this.put(IDENTITY_CERT_EXPIRY_THRESHOLD, "90");
+            this.put(SCA_X509_CERT_EXPIRY_THRESHOLD, "10");
```

Comment:
I wonder if this should be closer to the 90 days we use for the identity cert. Maybe bump it to 30 for the default and we can adjust accordingly later.